public/react/components/Article.js

```diff
@@ -2,15 +2,16 @@ import React from 'react'
 
 export const Article = ({ article }) => {
   //<h3>{article.authorId}</h3> need to get author name by author id
+  console.log('Article', article)
   return (
     <>
-      {console.log("hi")}
+      {article && <h2>An article</h2>}
```

Comment:
I'm using the `&&` operator to only show this `<h2>` if the article data is truthy, otherwise the `<h2>` doesn't show

You might prefer to extract this logic and do

```js
{articleData && <Article article={articleData} />}
```

in the `PagesList` component instead